test/api/api_test.go

```diff
@@ -189,37 +184,81 @@ func RunApiServer(t *testing.T, binary string, storageApiHost string) string {
 	return apiUrl
 }
 
+type ApiRequest struct {
+	Path   string      `json:"path"`
+	Method string      `json:"method"`
+	Body   interface{} `json:"body"`
```

Comment:
Tu by bolo mozno dobre mat nastavene aj `validate` tags,  a skontrolovat, ci je definicia validna, cez `validator`.
Chybova hlaska tak bude viac jasna, ako ked to spadne neskor.

Reply:
ok upraveno 👍